modules/structs/issue_label.go

```diff
@@ -11,6 +11,8 @@ type Label struct {
 	Name string `json:"name"`
 	// example: false
 	Exclusive bool `json:"exclusive"`
+	// example: false
+	IsArchived bool `json:"is_archived"`
```

Comment:
Ah, that's the location I commented about above.

Reply:
Same as above